spec/models/claim/advocate_claim_spec.rb

```diff
@@ -926,36 +926,36 @@
   end
 
   describe '#fixed_fees' do
-    let(:ct_fixed_1) { create(:case_type, :fixed_fee) }
-    let(:ct_fixed_2) { create(:case_type, :fixed_fee) }
-    let!(:claim_1) { create(:claim, case_type_id: ct_fixed_1.id) }
-    let!(:claim_2) { create(:claim, case_type_id: ct_fixed_2.id) }
+    let(:fixed_fee_case_type) { create(:case_type, :fixed_fee) }
+    let(:another_fixed_fee_case_type) { create(:case_type, :fixed_fee) }
+    let!(:first_claim) { create(:claim, case_type_id: fixed_fee_case_type.id) }
+    let!(:second_claim) { create(:claim, case_type_id: another_fixed_fee_case_type.id) }
 
     before do
       create(:claim, case_type_id: create(:case_type).id)
       create(:claim, case_type_id: create(:case_type).id)
     end
 
     it { expect(described_class.fixed_fee.count).to eq 2 }
-    it { expect(described_class.fixed_fee).to include claim_1 }
-    it { expect(described_class.fixed_fee).to include claim_2 }
+    it { expect(described_class.fixed_fee).to include first_claim }
+    it { expect(described_class.fixed_fee).to include second_claim }
   end
 
   describe '.total_greater_than_or_equal_to' do
-    let(:not_greater_than_400) do
+    let(:claims_with_total_under_400_pounds) do
       claims = []
 
-      [100, 200, 399, 2].each do |value|
+      [100, 200, 399, 399.99, 2].each do |value|
         claims << create(:submitted_claim, total: value)
       end
 
       claims
     end
 
-    let(:greater_than_400) do
+    let(:claims_with_total_greater_than_or_equal_to_400_pounds) do
```

Comment:
Not a blocker but a possible improvement. This could be done simpler using `each_with_object`.

Reply:
Good spot. I've pushed an extra commit to improve it (and also for `claims_with_total_under_400_pounds`, which I realised wasn't actually being used anywhere, so I've added an additional test for that scope too).